ui/apps/platform/src/Containers/Integrations/IntegrationTiles/IntegrationTilesPage.tsx

```diff
@@ -18,6 +19,8 @@ function IntegrationTilesPage(): ReactElement {
 
     return (
         <>
+            {/*TODO(ROX-25633): Remove the banner again.*/}
+            <OcmDeprecatedTokenBanner />
```

Comment:
I'd recommend moving this directly below the `<PageSection>` that contains the page title, and extracting the `<PageSection>` out of the banner and keeping it here with the other sections.